releasenotes/notes/0.6/feature-support-calibrations-roundtrip-47f09bd9ff803479.yaml

```diff
@@ -1,27 +1,27 @@
 ---
 features:
   - |
-    JSON data format is supoorted for saving :class:`.Calibrations` instance.
+    JSON data format is now supported for saving a :class:`.Calibrations` instance.
     This leverages a custom JSON encoder and decoder to serialize 
     the entire calibration data including user provided schedule templates.
     Output JSON data is formatted into the standard data model which is intentionally
-    agnostic to the calibration data structure, and this allows community
+    agnostic to the calibration data structure, which this allows community
     developer to reuse the calibration data in their platform.
     See :mod:`qiskit_experiments.calibration_management.save_utils` for data models.
```

Comment:
@nkanazawa1989 Should `calibration_management.save_utils` be exposed in the API docs? It seems to be internal-only. If so, we should remove the mention here.

Reply:
It says at the top of the file that it is not intended to be a stable user-facing API. It has to be stable to whatever commitment we make to loading old data though. The main question is should we document the format at all or expect users who care to look into the Python code themselves any way.

I would say just remove the reference here, but the previous sentence doesn't make sense if the model is not documented. I think I would just remove the previous sentence as well because we don't need to commit to supporting a calibration storage format. Others can still use it if they want to. What do you think @nkanazawa1989?

Reply:
I'm fine with removing the sentence.

Reply:
Done in b57deb0.

Reply:
I would remove the part or all of the previous sentence as well. Since the format is not documented, we don't really support users using it outside of the package (though they can try if they want).